### ISSUE TITLE
bb-context-menu does not have a label

### ISSUE THREAD

**Issue by Blackbaud-PaulCrowder** [blackbaud/skyux1#465]
### Expected behavior

The `bb-context-menu` should have an `aria-label` attribute for screen readers.
### Actual behavior

The `bb-context-menu` does not have an `aria-label` attribute.  This causes the Accessibility audit from Chrome's [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) to fail with [`AX_TEXT_01`](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01).
### Steps to reproduce
- Install Accessibility Tools and run an audit on a page with a `bb-context-menu` element.
- Note the violation.
### Plunker

http://plnkr.co/edit/gTbPn4KYEufJjRBneF7M?p=preview

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.5.15](https://github.com/blackbaud/skyux/releases/tag/v1.5.15)
